src/pages/patientView/trialMatch/TrialMatchTable.tsx

```diff
@@ -0,0 +1,236 @@
+            <div>
+                {trial.matches.map((armMatch: IArmMatch, index: number) => (
+                    <div>
+                        <div>
+                            <div>
+                                {armMatch.matches.map((clinicalGroupMatch: IClinicalGroupMatch, cgIndex:number) => (
+                                    <span className={styles.criteriaContainer}>
+                                        {this.getGenomicMatch(clinicalGroupMatch)}
+                                        {this.getClinicalMatch(clinicalGroupMatch)}
+                                        <If condition={cgIndex < armMatch.matches.length - 1}><hr className={styles.criteriaHr}/></If>
+                                    </span>
+                                ))}
+                            </div>
+                             <If condition={armMatch.armDescription !== ''}>
+                                <div className={styles.armDiv}>
+                                    <span>Arm: {armMatch.armDescription}</span>
+                                </div>
+                            </If>
+                            <If condition={armMatch.drugs.length > 0}>
+                                <div className={styles.armDiv}>
+                                    <span>Intervention: {armMatch.drugs.join(', ')}</span>
+                                </div>
+                            </If>
+                        </div>
+                        <If condition={index < trial.matches.length - 1}><hr className={styles.criteriaHr}/></If>
+                    </div>
+                ))}
+            </div>
+        ),
+        width: this.columnsWidth[ColumnKey.MATCHING_CRITERIA]
+    }];
+
+    public getClinicalMatch(clinicalGroupMatch: IClinicalGroupMatch) {
+        return (
+            <span className={styles.firstRight}>
+                <span className={styles.secondLeft}>{clinicalGroupMatch.trialAgeNumerical + ' yrs old'}</span>
+                <span className={styles.secondRight}>
+                    {clinicalGroupMatch.trialOncotreePrimaryDiagnosis.positive.join(', ')}
+                    {clinicalGroupMatch.trialOncotreePrimaryDiagnosis.negative.length > 0 &&
+                        <span>
+                            <b> except in </b>
+                            { clinicalGroupMatch.trialOncotreePrimaryDiagnosis.negative.length < 4 ? (
+                                <span>{clinicalGroupMatch.trialOncotreePrimaryDiagnosis.negative.join(', ').replace(/,(?!.*,)/gmi, ' and')}</span>
+                            ) : (
+                                <DefaultTooltip
+                                    placement='bottomLeft'
+                                    trigger={['hover', 'focus']}
+                                    overlay={this.tooltipClinicalContent(clinicalGroupMatch.trialOncotreePrimaryDiagnosis.negative)}
+                                    destroyTooltipOnHide={false}
```

Comment:
should probably be destroyTooltipOnHide=trye.  we don't want to keep lots of tooltip HTML in dom.  better just to regenerate it.

Reply:
Changed to true